base/stream.jl

```diff
@@ -433,30 +412,6 @@ end
 
 alloc_buf_hook(stream::LibuvStream, size::UInt) = alloc_request(stream.buffer, UInt(size))
 
-function notify_filled(buffer::IOBuffer, nread::Int, base::Ptr{Void}, len::UInt)
```

Comment:
documentation sure, exporting for one package that I think is mostly deprecated, not so sure - though I believe not all of its dependents have moved away from it yet, so probably shouldn't try to break it - especially not right now